src/sdk/pynni/nni/hyperband_advisor/hyperband_advisor.py

```diff
@@ -303,11 +307,6 @@ def __init__(self, R=60, eta=3, optimize_mode='maximize'):
 
     def handle_initialize(self, data):
         """data is search space
```

Comment:
why remove the docstring?

Reply:
Because they are wrong. Data should be a dict of search space.

Reply:
Actually I made that clear in MsgDispatcherBase.

Reply:
I think you could put the right docstring in the same place?